JavaScript/writable-enumerable-configurable.md

```diff
@@ -1,127 +1,163 @@
-```name: ‘Bart’``` property, we can type the following code:
-```
-console.log(Object.getOwnPropertyDescriptor(boy, ‘name’))
+
+However, there is more to the properties of an object than just a name and value. Properties also have the following attributes: `writable`, `enumerable` and `configurable`. To display, for instance, the attributes of the `name: 'Bart'` property, we can type the following code:
+
+```javascript
+console.log(Object.getOwnPropertyDescriptor(boy, 'name'))
 ```
+
 This will be displayed:
-```
+
+```javascript
 {
-value:	"Bart",
+value:	'Bart',
 writable:	true,
 enumerable:	true,
 configurable:	true
 }
 ```
-We can see that all three are set to ```true``` by default. Please remember that we are examining only the ```name``` here in our ```boy``` object.
+
+We can see that all three are set to `true` by default. Please remember that we are examining only the `name` here in our `boy` object.
+
 But what do they do? Let’s examine each one separately.
 
-WRITABLE:
+###Writable:
 
-The ```writable``` attribute determines whether we can change that property from its current value. Our initial value of ```name``` is ```Bart```.
+The `writable` attribute determines whether we can change a property from its current value. Our initial value of `name` is `'Bart'`.
 
-So long as ```writable``` is set to ```true```, we can change ```name``` as normal:
-```
-boy.name = ‘Leonard’;
-```
-However, let’s set ```writable``` to ```false```:
-```
-Object.defineProperty(boy, ‘name’, {writable: false});
-```
-If you now examine our ```name``` with the same code as before:
+So long as `writable` is set to `true`, we can change `name` as normal:
+
+```javascript
+boy.name = 'Leonard';
 ```
-console.log(Object.getOwnPropertyDescriptor(boy, ‘name’))
+
+However, let’s set `writable` to `false`:
+
+```javascript
+Object.defineProperty(boy, 'name', { writable: false });
 ```
-This will now be displayed:
+
+If we now examine our `name` with the same code as before:
+
+```javascript
+console.log(Object.getOwnPropertyDescriptor(boy, 'name'))
 ```
+
+The following will be displayed:
+
+```javascript
 {
-value:	"Bart",
+value: 'Bart',
 writable:	false,
 enumerable:	true,
 configurable:	true
 }
 ```
-Once you set ```writable``` of ```name``` to ```false```, and try to change the value of ```name```:
-```Boy.name = ‘Leonard’;```
 
-```name``` will still be set to ```Bart```.
+Once the `writable` attribute of the `name` property has been set to `false`, then trying to change the value of `name`:
 
-Of course, if you change the ```name``` before you set ```writable``` to ```false```, then that second name will now become ‘unchangeable’ so to speak.
+```javascript
+boy.name = 'Leonard';
+```
 
-WARNING!! You have to be very careful now because if the value of ```name``` is another object (so not a primitive like a string), for
-example: 
+Will result in `name`  still being set to `'Bart'`.
```

Comment:
Thanks for checking out the PR & for the comment. I think the latest push addresses this issue? Do let me know if not or if anything else could stand to be improved!